modules/telemetry-otel/src/test/java/org/opensearch/telemetry/OTelTelemetryModulePluginTests.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry;
+
+import org.opensearch.common.SuppressForbidden;
+import org.opensearch.common.settings.ClusterSettings;
+import org.opensearch.common.settings.Setting;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.common.util.FeatureFlags;
+import org.opensearch.telemetry.tracing.OtelTracingTelemetry;
+import org.opensearch.telemetry.tracing.TracingTelemetry;
+import org.opensearch.test.OpenSearchTestCase;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.Optional;
+
+import static org.opensearch.telemetry.OTelTelemetryModulePlugin.OTEL_TRACER_NAME;
+
+public class OTelTelemetryModulePluginTests extends OpenSearchTestCase {
```

Comment:
You may consider using `OpenSearchIntegTestCase` instead of manipulating system properties, it has `featureFlagSettings()`  you could apply to set feature flags in scope (see please `OpenSearchNetty4IntegTestCase` for example)l

Reply:
Not needed now